packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md

```diff
@@ -89,6 +89,8 @@ Each API is a link that navigates you to the [dashboard view of API rate limits
 
 You can perform a search in your table list by making an entry in the Search box. You can also perform a "lookup" with the Search. For example, you're calling Okta with an example `requestURI` and you want to know the rate limit that is applied to calls that are made against that `requestURI`.
 
+API endpoints in the dashboard that end with an asterisk (for example, `/api/v1/users/*`) generally refer to calls that use that base schema and count towards the overall limit, unless the endpoint is specifically called out in the dashboard. It's also important to note the HTTP operation involved with each call, as the same call with different operations can count towards different limits.
```

Comment:
```suggestion
API endpoints in the dashboard that end with an asterisk (for example, `/api/v1/users/*`) refer to calls that use that base schema and count towards the overall limit, unless the endpoint is specifically called out in the dashboard. It's also important to note the HTTP operation involved with each call, as the same call with different operations can count towards different limits.
```

Reply:
@susanharper-okta I added "generally" here to be purposefully vague. Support can configure things that can make this statement incorrect.